hikari/api/rest.py

```diff
@@ -5941,9 +5941,10 @@ async def ban_user(
 
         Other Parameters
         ----------------
-        delete_message_days : hikari.undefined.UndefinedOr[int]
-            If provided, the number of days to delete messages for.
-            This must be between 0 and 7.
+        delete_message_seconds : hikari.undefined.UndefinedNoneOr[hikari.internal.time.Intervalish]
```

Comment:
Not really deprecating it if you're still removing the arg from the rest client method since ppl are using this directly and this will still be a breaking change for them.

Reply:
Forgot to deprecate it there, will fix!

Reply:
Changed :)